_data/developer.yml

```
@@ -479,6 +479,12 @@ websites:
     facebook: saucelabs
     email_address: support@saucelabs.com
 
+  - name: Scripting Helpers
+    url: https://scriptinghelpers.org/
+    img: scriptinghelpers.png
+    tfa:
+      - software
```

```suggestion
      - totp
```
That would be totp now :)